src/components/Images.vue

```diff
@@ -298,30 +308,65 @@ export default {
     deleteImage(obj) {
       this.currentCommand = `delete ${ obj.imageName }:${ obj.tag }`;
       this.mainWindowScroll = this.$refs.fullWindow.parentElement.parentElement.scrollTop;
+      this.postOpSuccessHandler = this.postDeleteSuccessHandler;
       this.startRunningCommand('delete');
       ipcRenderer.send('confirm-do-image-deletion', obj.imageName.trim(), obj.imageID.trim());
     },
+    postDeleteSuccessHandler() {
+      if (this.imageManagerOutput === '') {
+        this.closeOutputWindow(null);
+      }
+    },
     doPush(obj) {
       this.currentCommand = `push ${ obj.imageName }:${ obj.tag }`;
+      this.mainWindowScroll = this.$refs.fullWindow.parentElement.parentElement.scrollTop;
       this.startRunningCommand('push');
       ipcRenderer.send('do-image-push', obj.imageName.trim(), obj.imageID.trim(), obj.tag.trim());
     },
     doBuildAnImage() {
       this.currentCommand = `build ${ this.imageToBuild }`;
       this.fieldToClear = 'imageToBuild';
+      this.postCloseOutputWindowHandler = this.scrollToPulledImage(this.imageToBuild);
       this.startRunningCommand('build');
       ipcRenderer.send('do-image-build', this.imageToBuild.trim());
     },
     doPullAnImage() {
       this.currentCommand = `pull ${ this.imageToPull }`;
       this.fieldToClear = 'imageToPull';
+      this.postCloseOutputWindowHandler = this.scrollToPulledImage(this.imageToPull);
       this.startRunningCommand('pull');
       ipcRenderer.send('do-image-pull', this.imageToPull.trim());
     },
+    scrollToPulledImage(imageToPull) {
+      return () => {
+        if (!this.imageManagerOutput.trimStart().startsWith('Error:')) {
+          //  error on pull: go back stay here
+          const [imageName, possibleTag] = imageToPull.split(':', 2);
+          const tag = possibleTag || 'latest';
+          const rows = document.querySelector('table.sortable-table').rows;
```

Comment:
`this.$refs` doesn't contain a reference to a `ref` attribute on the `SortableTable`. I couldn't find why, but think that it's because you can't have a `ref` on a component, only on an HTML tag. The `ref` attr most likely gets lost during component processing.